### ISSUE TITLE
Checkbox checkmark invisible in nightly

### ISSUE THREAD

**Issue by freaktechnik** [mozilla/multi-account-containers#1306]
### Actual behavior
Checkmark in checkbox to always open in container in popup is not loaded in Firefox Nightly

### Expected behavior
Checkmark is visible.

### Steps to reproduce
1. Open MAC popup for a page that's loaded in acontainer
2. Toggle "Always open in this container"

**Comment by cpeterso** [comment]
I first see this checkbox problem in Firefox 64 Beta. The checkbox's functionality works, but the visual indicator doesn't show up.

1. If I uninstall and install MAC in Firefox 63, the check is visible.
2. If I uninstall and install MAC in Firefox 64 Beta or 65 Nightly, the check is not visible.
3. If I _then_ load the same user profile in Firefox 63, the check is not visible! Installing in 64+ breaks some setting that causes the check to stop working even in 63.

I am using MAC version 6.0.0.

**Comment by dimtion** [comment]
Same here for Firefox 64 Beta, the check-mark is not visible but the checkbox's functionality works. On Firefox 63 the issue does not arises.

System Information:
- MAC 6.0.0
- Firefox 64b
- MacOS 10.14

**Comment by luixxiul** [comment]
Hello, when you click the addon icon on the toolbar, this error message is displayed on the browser console:

````
Error: Invalid badge background color: "" ExtensionUtils.jsm
````

I'm not sure if the error is related with this issue though. Thanks,

**Comment by llostris** [comment]
It's also invisible in Firefox 63.0.3 which I believe is latest stable version.

**Comment by Ambient-Impact** [comment]
I'm also seeing this same issue on both stable and Developer Edition 64.0b14. I only just realized that the checkbox was taking effect but not displaying its state, thought it was disabled for weeks. D:

**Comment by psylenced** [comment]
This has been frustrating me too since b64 as I thought I've been unable to edit new sites due to the box being disabled.

Didn't realise it still worked and held state even though it wasn't showing the change visibly.

**Comment by Gertbucket** [comment]
What's the hold up? This is a critical bug.

**Comment by groovecoder** [comment]
Most of us maintainers have been in heavy travel & work-week activities but we are planning to fix and release ASAP.

**Comment by cpeterso** [comment]
The checkbox's check is working correctly for me as of extension version 6.0.1. Thanks!

**Comment by jillesvangurp** [comment]
Still broken for me with 6.0.1. I'm running the mac version of 65.0b4. I just checked that everything is updated.

**Comment by luixxiul** [comment]
This looks solved on the latest beta, 65.0b4.

**Comment by freaktechnik** [comment]
Sam, checkboxes work properly again on nightly. Taking the freedom to close this.
